plugins/by-name/wrapping/default.nix

```
@@ -18,9 +14,68 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
       By default, wrapping.nvim will output a message to the
       command line when the hard or soft mode is set.
     '';
+
+    create_commands = defaultNullOpts.mkBool true ''
+      If true, the plugin will create the following
+      commands to set/override a wrapping mode in case
+      it is not autodetected correctly:
+
+      * HardWrapMode
+
+      * SoftWrapMode
+
+      * ToggleWrapMode
+    '';
+
+    create_keymaps = defaultNullOpts.mkBool true ''
+      If true, it will create the following normal-mode keymappings:
+
+      * [ow (soft wrap mode)
```

```suggestion
      * `[ow` (soft wrap mode)
```

Another example :slightly_smiling_face: